src/WebJobs.Script.WebHost/Models/HostAssignmentContext.cs

```diff
@@ -129,6 +129,23 @@ public void ApplyAppSettings(IEnvironment environment)
                     environment.SetEnvironmentVariable(EnvironmentSettingNames.CorsAllowedOrigins, allowedOrigins);
                 }
             }
+
+            if (EasyAuthSettings != null)
+            {
+                // App settings take precedence over site config for easy auth enabled.
+                if (string.IsNullOrEmpty(environment.GetEnvironmentVariable(EnvironmentSettingNames.EasyAuthEnabled)))
+                {
+                    logger.LogWarning($"ApplyAppSettings is adding {EnvironmentSettingNames.EasyAuthEnabled} = {EasyAuthSettings.SiteAuthEnabled.ToString()}");
+                    environment.SetEnvironmentVariable(EnvironmentSettingNames.EasyAuthEnabled, EasyAuthSettings.SiteAuthEnabled.ToString());
+                }
+                else
+                {
+                    logger.LogWarning($"ApplyAppSettings operating on existing {EnvironmentSettingNames.EasyAuthEnabled} = {environment.GetEnvironmentVariable(EnvironmentSettingNames.EasyAuthEnabled)}");
```

Comment:
Assuming you added logs here for debugging? The last log should be enough. If you do need to keep rest of the logs, change them to Trace level.